flood_api/models/detailed_types.py

```diff
@@ -1,23 +1,22 @@
-from datetime import datetime
+from datetime import date
 from typing import List
 
 from pydantic import BaseModel, Field
 
-from flood_api.models.shared_types import Feature, FeatureCollection
+from flood_api.models.shared_types import CustomBaseModel, Feature, FeatureCollection
 
 
-class DetailedProperties(BaseModel):
-    issued_on: datetime = Field(
+class DetailedProperties(CustomBaseModel):
+    issued_on: date = Field(
         ...,
         description="The date the detailed forecast was issued on.",
```

Comment:
Which timezone is this? This needs to be documentet

Reply:
Done